File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/Spade.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Sword;

/* JADX INFO: loaded from: classes3.dex */
public class Spade extends Sword {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_sword_spade_name;
        this.idDescription = R.string.weapon_sword_spade_description;
        this.idImage = R.drawable.spade;
        this.price = 0L;
        this.constitution = 1;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/Spade.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Sword

class Spade : Sword() {
    override fun configureProperties() {
        idName = R.string.weapon_sword_spade_name
        idDescription = R.string.weapon_sword_spade_description
        idImage = R.drawable.spade
        price = 0L
        constitution = 1
    }
}
